package.json

```diff
@@ -112,10 +112,10 @@
     "url": "https://github.com/coryhouse/react-slingshot"
   },
   "jest": {
-    "moduleNameMapper": {
-      "\\.(css|scss)$": "identity-obj-proxy",
-      "^.+\\.(gif|ttf|eot|svg|woff|woff2|ico)$": "<rootDir>/tools/fileMock.js"
-    }
+    "moduleNameMapper": { 
+    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/tools/assetsTransformer.js", 
+    "\\.(css|less)$": "<rootDir>/tools/assetsTransformer.js" 
```

Comment:
No worries I made these changes directly to master. The not regex did not work as I had intended.